### ISSUE TITLE
ACM dimensions are not mapped

### ISSUE THREAD

**Issue by VladimirAlexiev** [american-art/acm#12]
http://data.americanartcollaborative.org/page/acm/object/1980_49 has quite little data.
In particular, dimensions are not mapped.
The source includes dimension data as follows (weird newline-separated parallel arrays):
```xml
    <atom name="MedType_tab" type="text" size="short">Height
Width</atom>
    <atom name="MedUnit_tab" type="text" size="short">inches
inches</atom>
    <atom name="MedUnitMetric_tab" type="text" size="short">cm
cm</atom>
    <atom name="MedValue_tab" type="text" size="short">11 1/4
8 3/4</atom>
    <atom name="MedValueMetric_tab" type="text" size="short">28.58
22.23</atom>
```
This should be mapped to Display dimension and Structured dimension, see https://github.com/american-art/GM/issues/17 for links. This below follows that model, just is a bit more specific (@workergnome, @azaroth42 do you agree with it).

You need to reformat the source data quite a lot, eg
```ttl
<acm/object/1980_49>
  crm:P67i_is_referred_to_by <acm/object/1980_49/dimension>;
  crm:P43_has_dimension
    <acm/object/1980_49/height/inches>,
    <acm/object/1980_49/width/inches>,
    <acm/object/1980_49/height/cm>,
    <acm/object/1980_49/width/cm>.

<acm/object/1980_49/dimension> a crm:E33_Linguistic_Object;
  crm:P2_has_type aat:300266036;
  rdf:value """
Height: 11 1/4 inches, 28.58 cm
Width:  8 3/4  inches, 22.23 cm
""".

<acm/object/1980_49/height/inches> a crm:E54_Dimension; crm:P2_has_type aat:300055644; crm:P91_has_unit aat:300379100; rdf:value "11 1/4".
<acm/object/1980_49/width/inches>  a crm:E54_Dimension; crm:P2_has_type aat:300055647; crm:P91_has_unit aat:300379100; rdf:value "8 3/4".
<acm/object/1980_49/height/cm>     a crm:E54_Dimension; crm:P2_has_type aat:300055644; crm:P91_has_unit aat:300379098; rdf:value 28.58.
<acm/object/1980_49/width/cm>      a crm:E54_Dimension; crm:P2_has_type aat:300055647; crm:P91_has_unit aat:300379098; rdf:value 22.23.

aat:300055644 skos:prefLabel "height".
aat:300055647 skos:prefLabel "width".
aat:300379100 skos:prefLabel "inches".
aat:300379098 skos:prefLabel "cm".
```
- map other types that you find also to AAT.
- ideally, the skos:prefLabels should be emitted only once. Easiest is to make a turtle manually, as you do the prev bullet
- would be nice to mark the dimensions as Metric/Imperial but AAT doesn't have such concepts. Should we ask Patricia to add them?
- please note we've emitted Imperial values as string but Metric values as xsd:decimal. This will allow faster comparison of these numbers.

**Comment by workergnome** [comment]
That mapping looks good to me.  Explicit units are required if we're going to do anything with these numbers beyond display them.

(as an aside to @VladimirAlexiev:  Rather than go through the bother of trying to define everything on AAT, it might be more useful to try and crosswalk as much as possible between AAT and QUDT.  Having the explicit conversion factors, and the types, and all of the conversion logic are really useful when I want to start treating the dimensions as semantically useful, and it seems a waste of time to duplicate that entirely in AAT when someone's already done it.)

**Comment by VladimirAlexiev** [comment]
You need to display the units (or the E33 display dimension, which has all).
QUDT is good: AAT has very few units. But even QUDT doesn't have all units needed for CH.

Doing math with Imperial is a royal pain. Not only because of the natural fractions, but also see recent CRM discussions (with @azaroth42) "how would CRM represent a dimension of 1 ft 2 in?" All museums I've seen that emit Imperial, also emit Metric.

**Comment by KatherineMoloney** [comment]
We do have Metric dimensions in our records. Is this something I can amend while validating?

**Comment by workergnome** [comment]
Metric units would be helpful, if you have them and ISI is willing to add them to the mapping.

**Comment by VladimirAlexiev** [comment]
@KatherineMoloney Metric measurements are in the source data, see `MedValueMetric_tab, MedUnitMetric_tab` above. 
The layout is weird newline-separated parallel arrays, so @bsnikhila will have a "fun" time parsing that.

**Comment by bsnikhila** [comment]
Unfortunately, it will not be possible to map the dimensions now. If the data is cleaned up a little, then the dimensions can be mapped.
